pytometry/plotting/_scatter_density.py

```diff
@@ -9,18 +12,18 @@
 
 def scatter_density(
     adata: AnnData,
-    x="FSC-A",
-    y="SSC-A",
-    x_label="FSC-A",
-    y_label="SSC-A",
-    x_scale="linear",
-    y_scale="linear",
-    x_lim=[-2 * 1e4, 3 * 1e5],
-    y_lim=[-2 * 1e4, 3 * 1e5],
-    ax=None,
-    cmap="jet",
-    vmin=None,
-    vmax=None,
+    x: str = "FSC-A",
+    y: str = "SSC-A",
+    x_label: str = "FSC-A",
+    y_label: str = "SSC-A",
+    x_scale: str = "linear",
+    y_scale: str = "linear",
+    x_lim: List[float] = [-2 * 1e4, 3 * 1e5],
+    y_lim: List[float] = [-2 * 1e4, 3 * 1e5],
+    ax: Optional[mpl.Axes] = None,
```

Comment:
I think that you can do `from matplotlib.axes import Axes` and the replace `mpl.Axes` by `Axes`